src/hooks/matomoHooks.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+import { useMatomo } from 'matomo-tracker-react-native';
+
+/**
+ * Tracks app start as action with prefixed 'App' category
+ */
+export const useMatomoTrackAppStart = () => {
```

Comment:
we do not need this as extra hook, as we only use it once in App.js

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/109/commits/2fbd5922b5615a5c27dfcb8e0659ae9f101da39b